docs/multi-cluster-install.md

```diff
@@ -57,12 +57,13 @@ If you have `jq` and `base64` available then this one-liners will pull all CA ce
 `${HOME}/.kube/config` and place then in a file named `ca.pem`.
 
 ```shell
-kubectl config view -o json --raw  | jq -r '.clusters[].cluster["certificate-authority-data"]' | base64 -d > ca.pem
+# replace CLUSTERNAME with the name of the cluster according to your ${HOME}/.kube/config
+kubectl config view -o json --raw  | jq -r '.clusters[] | select(.name=="CLUSTERNAME").cluster["certificate-authority-data"]' | base64 -d > ca.pem
 ```
 
 ## Install
 
-In the following example it will be assumed the API server URL is `https://example.com:6443`
+In the following example it will be assumed the API server URL from the `${HOME}/.kube/config` which is `https://example.com:6443`
```

Comment:
`${HOME}/.kube/config` is the default location but it can live in anywhere. Might just want to mention something generic like kubeconfig file.

Reply:
True, I will change the text